doc/build-helpers/fetchers.chapter.md

```diff
@@ -80,6 +80,10 @@ stdenv.mkDerivation {
 
 The main difference between `fetchurl` and `fetchzip` is in how they store the contents. `fetchurl` will store the unaltered contents of the URL within the Nix store. `fetchzip` on the other hand, will decompress the archive for you, making files and directories directly accessible in the future. `fetchzip` can only be used with archives. Despite the name, `fetchzip` is not limited to .zip files and can also be used with any tarball.
 
+- `downloadToTemp`: defaults to false. If true, saves the source under
+  `$TMPDIR/file`, to be used in conjunction with `postFetch`...
+- `postFetch`: postprocess the downloaded archive
```

Comment:
Can you add some more text explaining that these are only applicable to `fetchurl`, and not `fetchzip`? The section this text is in talks about both, and the paragraph just before this ends talking about `fetchzip`, so this new text is definitely confusing as-is.

Regarding the documentation for `downloadToTemp`, I think documenting that it saves to `$TMPDIR/file` might not be a good idea to avoid people directly using that value in their code. It might be best to document that the temporary file path will be at `$downloadedFile`.
Also, can you add more text explaining `postFetch`? What type is it? How do you use it? The documentation in the code for `postFetch` is more descriptive, you can use it as a starting point.